recipe/meta.yaml

```diff
@@ -1,8 +1,7 @@
 {% set version = "5.0.0" %}
 {% set sha256 = "019f23c2192df793ac746595e94a403908749f8e0c484b403476d2611dd20970" %}
 
-{% set clang_variant = os.environ.get('CLANG_VARIANT', 'default') %}
-{% set build_number = "0" %}
+{% set build_number = "1000" %}
```

Comment:
I'm surprised this is still happening. What benefit does it still offer?

Reply:
The `build_number = "1000"` statement is for the new compilers rebuild of conda-forge.